CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # edge-react-gui
 
+## 2.13.0 (2022-04-11)
+
+- Add Tomb Finance's Masonry and Cemetery platforms
+- New swap provider SpookySwap
+- Upgrade React Native to v0.67.2
+- Convert wallet list scenes and components to use React hooks
+- Overlay parent currency icon on top of token icons
+- New animated sync circle
+- Refactor wallet list components to remove `GuiWallet` and unnecessary renders
+- Fix wallet list swipe button navigation
```

Comment:
This is a bit of an understatement. 
It should say:
- New animated wallet swipe list/row/buttons